components/Spotlight.js

```diff
@@ -1,14 +1,21 @@
 import Image from "next/image";
+import styled from "styled-components";
+
+const Container = styled.div`
```

Comment:
As you prefix your other styled components with "styled", I suggest to do the same here for consistency.